addons/storyshots/storyshots-core/src/frameworks/frameworkLoader.ts

```diff
@@ -13,13 +13,7 @@ function getLoaders(): Loader[] {
     .readdirSync(__dirname)
```

Comment:
When this code actually executes, `__dirname` is '<package-root>/dist/frameworks', so there are always compiled `__dirname/<framework>/loader.js` files available